integreat_cms/cms/forms/regions/region_form.py

```diff
@@ -27,6 +27,28 @@
 logger = logging.getLogger(__name__)
 
 
+class CheckboxSelectMultipleWithDisabled(forms.CheckboxSelectMultiple):
+    """
+    This class adds functionality for disabling certain choices in the
+    :class:`~django.forms.CheckboxSelectMultiple` widget
+    """
+
+    disabled_options = []
+
+    def create_option(self, *args, **kwargs):
+        """
+        Overwrites the parent's method in order to disable
+        a set of pre-determined options
+
+        :return: a single option
+        :rtype: dict
+        """
+        option = super().create_option(*args, **kwargs)
+        if option["value"] in self.disabled_options:
+            option["attrs"]["disabled"] = True
```

Comment:
```suggestion
            option["attrs"]["class"] = "fake-disable"
```
`disabled = True` prevents the field from being submitted, so this would be an alternative (though not really satisfying, since it's still possible to disable the field, I guess because of the label tag)...

Reply:
I guess one solution would be to add
```
label:has(> .fake-disable) {
    @apply text-gray-600 pointer-events-none;
}
```
to our style sheet, but this only works in firefox if users enabled `layout.css.has-selector.enabled` in their config...
Not ideal, but maybe better than nothing?

Reply:
Deja-vu. RIght, we've had this topic before, with the HIX/MT interaction 😄 

Back then we resorted to JS, so I guess I'll do the same here.